app/pages/ServiceDiscoveryResults/ServiceDiscoveryResults.tsx

```diff
@@ -148,6 +150,27 @@ const InnerServiceDiscoveryResults = ({
     sortAlgoliaSubcategoryRefinements,
   } = category;
 
+  const configureProps: Record<string, any> = {
+    filters: `categories:'${algoliaCategoryName}'`,
+  };
+  if (!category.disableGeoLocation) {
+    Object.assign(configureProps, {
+      aroundLatLng: `${location.lat}, ${location.lng}`,
+      aroundRadius: searchRadius,
+      aroundPrecision: 1600,
+    });
+  }
+  if (whitelabel.useBoostedCategories) {
+    const selectedSubcategories = searchState.refinementList?.categories ?? [];
+    configureProps.optionalFilters = subcategoryNames
+      .filter((x) => selectedSubcategories.includes(x))
+      .map((x) => `boosted_category:${x}`)
+      // Our Algolia plan only allows for up to one optional filter, so just
+      // drop all but the first.
+      .slice(0, 1);
+    console.log(configureProps);
```

Comment:
Did you intend to leave in this console.log?

Reply:
Whoops, nice catch! Just removed it in https://github.com/ShelterTechSF/askdarcel-web/pull/1392/commits/46d2cebec68a39edec210e52da2c61f0e8199e06.